static/specification.html

```diff
@@ -98,35 +98,29 @@ <h2>
         Terminology
       </h2>
       <dl>
-        <dt>
-          <dfn>User</dfn>:
-        </dt>
-        <dd>
-          the party who is accessing the web-monetized website.
-        </dd>
-        <dt>
-          <dfn>Website</dfn>:
-        </dt>
-        <dd>
-          the web-monetized website.
-        </dd>
         <dt>
           <dfn data-lt="provider">Web Monetization provider</dfn>:
         </dt>
         <dd>
-          the party making payments on behalf of the user.
+          The party making payments on behalf of the user. A Web Monetization
+          provider uses the [[[SPSP]]] (<abbr title=
+          "Simple Payment Setup Protocol">SPSP</abbr>) on top of the
+          [[[Interledger]]] (<abbr title=
+          "Interledger Protocol">ILP</abbr>)/[[[STREAM]]] to provide a
+          high-level way to send money and data.
         </dd>
         <dt>
           <dfn>Web Monetization receiver</dfn>:
         </dt>
         <dd>
-          the party receiving payments on behalf of the website.
+          The party receiving payments on behalf of the website.
         </dd>
         <dt>
-          <dfn>User agent</dfn>:
+          <dfn>Payment end-point</dfn>:
```

Comment:
This is what we're calling both the "pointer" and the "href"? Should we keep this as the href, and add a separate entry for the payment pointer?

Reply:
Agree... although, I think this one is ok, as the spec explicitly states that the `href` must be a "payment end-point" (which is a SPSP end-point).

Reply:
Great!